pkgs/top-level/python-packages.nix

```diff
@@ -2469,6 +2469,10 @@ self: super: with self; {
   # Current latest
   django_4 = callPackage ../development/python-modules/django/4.nix { };
 
+  django-admin-sortable2_1 = callPackage ../development/python-modules/django-admin-sortable2/1.nix { };
+
+  django-admin-sortable2_2 = callPackage ../development/python-modules/django-admin-sortable2/2.nix { };
```

Comment:
We can only add one version which is likely the newest.

Reply:
I initially packaged both because I was interested in using it with Django 3 and I thought I might as well package the latest version. ;) I guess I can just use a package override in my project.